scripts/040-python.sh

```diff
@@ -1,19 +1,24 @@
 #!/bin/bash
 set -eux -o pipefail
 
-# Package install
+user=$(id -un 1000)
+home=/home/$user
+
+# Package install for python libraries
 apt-get -y install \
-    python3-pip \
     cmake \
     build-essential
 
-pip3 install --upgrade pip setuptools wheel
-
-# Install base python packages
-pip3 install -r /tmp/packer/python/requirements.txt
+# Package install for pyenv
+apt-get -y install \
+    libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev llvm \
+    libncurses5-dev libncursesw5-dev xz-utils tk-dev libgdbm-dev lzma lzma-dev \
+    tcl-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev wget curl \
+    make build-essential openssl
 
-# Install additional useful libraries for the competitors
-pip3 install -r /tmp/packer/python/libraries.txt
+mv /tmp/packer/python/install-pyenv.sh /tmp/install-pyenv.sh
+mv /tmp/packer/python/install-packages.sh /tmp/install-packages.sh
+chmod +x /tmp/install-pyenv.sh /tmp/install-packages.sh
 
-# Make the entrypoint folder
-mkdir -p /home/robot/.local/bin
+su - $user -c /tmp/install-pyenv.sh
+su - $user -c /tmp/install-packages.sh
```

Comment:
Suggestion: Instead of copying the files in and running directly, why not just run them from where they are?

```suggestion
su - $user -c /tmp/packer/python/install-pyenv.sh
su - $user -c /tmp/packer/python/install-packages.sh
```

And, best just make the files executable in the repo rather than needing to `chmod`.

Reply:
There are issues around relying on files being executable in the repos with windows development, but we can run the scripts where they are.
9f27446d064009d84451594018020a02c7b2c7d3